File: cliente-service/src/main/java/com/jmlq/cliente_service/model/Cliente.java
package com.jmlq.cliente_service.model;

import jakarta.persistence.*;
import lombok.*;

/**
 * Hereda de Persona.
 * Usa la misma PK (persona_id) como FK a persona.id.
 */
@Entity
@Table(name = "cliente")
@PrimaryKeyJoinColumn(name = "persona_id", referencedColumnName = "id")
@Getter
@Setter
@NoArgsConstructor
public class Cliente extends Persona {

    @Column(nullable = false)
    private String contrasena;

    @Column(nullable = false)
    private Boolean estado;
}

File: cloud/src/jmlq.nttdata.test.Domain/Cliente.cs
namespace jmlq.nttdata.test.Domain;

public class Cliente
{
    public Guid Id { get; set; }
    public string Nombre { get; set; }
    public string Identificacion { get; set; }
    public string Contrasena { get; set; }
    public bool Estado { get; set; }
}